.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+## This is my intent (chose one)
+
+- [ ] I want to report a bug
+- [ ] I want to request a feature or change
+- [ ] I want to provide or change a feature
+
+## The problem
+
+*describe the problem you encountered here. Could be a bug, a missing feature, a question that needs discussing…*
```

Comment:
@ZauberNerd suggested to add these hints as html comments. In that way they would not show up if someone forgets to delete them. 

```
<!-- describe the problem you encountered here. Could be a bug, a missing feature, a question that needs discussing… -->
```